src/view3d.cpp

```diff
@@ -311,37 +461,42 @@ void View3D::paintGL()
 //        s[barSize] = '|';
 //    tfm::printfln("%12f %4d %s", quality, frameTime, s);
 
-    m_incrementalFramebuffer->release();
-    QGLFramebufferObject::blitFramebuffer(0, QRect(0,0,width(),height()),
-                                          m_incrementalFramebuffer.get(),
-                                          QRect(0,0,width(),height()),
-                                          GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
+    // TODO: this should really render a texture onto a quad and not use glBlitFramebuffer
```

Comment:
Technically, it could probably be faster, but the tests I have read in the past claim it is slower (not a huge difference, though, maybe 20 microseconds). We should probably test this on current drivers with targeted hardware if we feel there is need for this to change.